src/test/java/com/google/javascript/cl2dts/provide_single_class.js

```diff
@@ -32,6 +32,11 @@ foo.bar.Baz.prototype.equals = function(b) {
   return false;
 };
 
+/**
+ * @private
+ */
+foo.bar.Baz.prototype.thisIsPrivate_ = function() {};
```

Comment:
add @private and @constructor to match the closure lib case. Easy to do since no usage needed :)